indra/tests/test_sources/__init__.py

```diff
@@ -0,0 +1,3 @@
+# -*- coding: utf-8 -*-
+
+"""A submodule for organizing tests for sources."""
```

Comment:
I think this is good though I'm not sure whether we should now move all the existing tests in here.

Reply:
For future PRs for sources, this might make organization easier. I wouldn't say we need to move other tests over at the moment though